### PR TITLE
Update Rule12-5.md

### PR DESCRIPTION
**Pull request by xingcx**

### CODE REVIEW

**Comment by JasonGlazer** [comment]
Original discussion at the link below. Please give this link time to load. This page might take a few seconds to load and another 4-8 seconds to land at the correct comment:

https://github.com/pnnl/ruleset-checking-tool/pull/19#discussion_r614289436

File: docs/section12/Rule12-5.md [hunk 1]
```diff
@@ -18,21 +18,19 @@
...
+- For each thermal_block from building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```  
+
+- For each zone from thermal block: ```zone_user in thermal_block_user.zones:```  
+
+- For each space from thermal zone: ```space_user in zone_user.spaces:```  
+
+  - Get the total miscellaneous_equipment in the space: ```space_total_misc_equipment_user = sum( equipment.peak_usage for equipment in space_user.miscellaneous_equipments )```  
+
+  - Get matching space from Proposed RMR: ```space_proposed = match_data_element(P_RMR, spaces, space_user.name)```  
+
```
Comment (supriyagoel) [inline docs/section12/Rule12-5.md:33]: we should now use ID for matching.

File: docs/section12/Rule12-5.md [hunk 2]
```diff
@@ -18,21 +18,19 @@
...
-    - Get the name of the space: ```space_name = space.name```
-    - Get the receptacle load in the space: ```space_receptacle = space.equipment_internal_gains```
-    - Save the receptacle load for all spaces in the Proposed model: ```space_receptacles_proposed[space_name] = space_receptacle```
-
-**Rule Assertion:** ```space_receptacles_user == space_receptacles_proposed```  
+- **Applicability Check 1:** ```if (rule-12-3.status == TRUE):```
+- Get the receptacle load of each space in the building segment in the User model: ```for building_segment_user in U_RMR.building.building_segments:```  
+
+- For each thermal_block from building segment: ```thermal_block_user in building_segment_user.thermal_blocks:```  
+
```
Comment (supriyagoel) [inline docs/section12/Rule12-5.md:25]: @xingcx should remove thermal block.

**Review by supriyagoel — CHANGES_REQUESTED** [review]

File: docs/section12/Rule12-5.md (hunk 2 above)
Comment (xingcx) [reply docs/section12/Rule12-5.md:25]: Updated.

File: docs/section12/Rule12-5.md (hunk 1 above)
Comment (xingcx) [reply docs/section12/Rule12-5.md:33]: Updated.

**Comment by mcneillj** [comment]
Archived. No longer needed.

**Pull request closed without merging**